### PR TITLE
Test/ldg 304 klaytn app implement verifysignature in python t

### PR DESCRIPTION
**Pull request by keiff3r**

### CODE REVIEW

File: .vscode/settings.json
Comment (Z4karia) [inline .vscode/settings.json:1]: to remove

File: .vscode/extensions.json
Comment (Z4karia) [inline .vscode/extensions.json:1]: to remove

File: Makefile [hunk 1]
```diff
@@ -34,12 +34,12 @@ ifeq ($(TARGET_NAME), TARGET_NANOX)
...
 endif
-APP_LOAD_PARAMS += --path "44'/501'"
+APP_LOAD_PARAMS += --path "44'/8217'"
 APP_LOAD_PARAMS += $(COMMON_LOAD_PARAMS)
 
-APPNAME      = "Solana"
-APPVERSION_M = 1
-APPVERSION_N = 3
+APPNAME      = "Klaytn"
+APPVERSION_M = 0
```
Comment (Z4karia) [inline Makefile]: 1.0.0

**Review by Z4karia — CHANGES_REQUESTED** [review]

**Comment by keiff3r** [comment]
The appropriate changes were made

**Pull request merged**